install/certmonger/dogtag-ipa-ca-renew-agent-submit

```diff
@@ -507,6 +508,8 @@ def main():
         certs.renewal_lock.release()
         shutil.rmtree(tmpdir)
 
+    api.Backend.ldap2.disconnect()
```

Comment:
You may want to move the disconnect() call in the finally block to make sure it gets executed